src/components/Section/Foreign/Activities/RealEstateActivity/RealEstateInterest.jsx

```diff
@@ -43,95 +27,98 @@ export default class RealEstateInterest extends ValidationElement {
       Sold: this.props.Sold,
       SoldNotApplicable: this.props.SoldNotApplicable,
       CoOwners: this.props.CoOwners,
-      ...queue
+      ...queue,
     })
   }
 
-  updateInterestTypes(values) {
-    let interestType = values.value
-    let selected = [...((this.props.InterestTypes || {}).values || [])]
+  updateInterestTypes = (values) => {
+    const interestType = values.value
+    const selected = [...((this.props.InterestTypes || {}).values || [])]
     if (selected.includes(interestType)) {
       selected.splice(selected.indexOf(interestType), 1)
     } else {
       selected.push(interestType)
     }
 
     this.update({
-      InterestTypes: { values: selected }
+      InterestTypes: { values: selected },
     })
   }
 
-  updateRealEstateType(values) {
+  updateRealEstateType = (values) => {
     this.update({
-      RealEstateType: values
+      RealEstateType: values,
     })
   }
 
-  updateAddress(values) {
+  updateAddress = (values) => {
     this.update({
-      Address: values
+      Address: values,
     })
   }
 
-  updateAcquired(values) {
+  updateAcquired = (values) => {
     this.update({
-      Acquired: values
+      Acquired: values,
     })
   }
 
-  updateHowAcquired(values) {
+  updateHowAcquired = (values) => {
     this.update({
-      HowAcquired: values
+      HowAcquired: values,
     })
   }
 
-  updateCost(values) {
+  updateCost = (values) => {
     this.update({
-      Cost: values
+      Cost: values,
     })
   }
 
-  updateCostEstimated(values) {
+  updateCostEstimated = (values) => {
     this.update({
-      CostEstimated: values
+      CostEstimated: values,
     })
   }
 
-  updateSold(values) {
+  updateSold = (values) => {
     this.update({
-      Sold: values
+      Sold: values,
     })
   }
 
-  updateSoldNotApplicable(values) {
+  updateSoldNotApplicable = (values) => {
     this.update({
-      SoldNotApplicable: values
+      Solid: values.applicable ? this.props.Solid : {},
```

Comment:
typo! 😂 `Solid -> Sold`

Reply:
...what on earth was going on with my brain when writing this 🙁